keyboards/keychron/c1/rules.mk

```diff
@@ -0,0 +1 @@
+OPENRGB_ENABLE = yes
```

Comment:
please don't add OpenRGB stuff here
additionally, this file should now be safe to be removed since you have a `keyboard.json`
```suggestion
```